namerd/core/src/main/scala/io/buoyant/namerd/InterfaceConfig.scala

```diff
@@ -21,6 +26,10 @@ abstract class InterfaceConfig extends PolymorphicConfig {
   )
 
   @JsonIgnore
+  @silent
+  def tlsParams = tls.map(_.params(None, LegacyKeyServerEngineFactory)).getOrElse(Stack.Params.empty)
```

Comment:
Does this work for the mesh interface?  I would have thought that `Some(Seq(ApplicationProtocolNames.HTTP_2))` would need to be specified for the `alpnProtocols` instead of `None`.